configs/orchestrator/cluster-install-configs/aws/aws.lokocfg

```diff
@@ -0,0 +1,33 @@
+variable "cluster_name" {}
+variable "ssh_pubkey" {}
+variable "route53_zone" {}
+variable "route53_zone_id" {}
+variable "aws_access_key_id" {}
+variable "aws_secret_access_key" {}
+variable "region" {}
+
+cluster "aws" {
+  asset_dir        = "./assets"
+  cluster_name     = var.cluster_name
+  controller_count = 1
+  dns_zone         = var.route53_zone
+  dns_zone_id      = var.route53_zone_id
+  region           = var.region
+  enable_csi       = true
+  ssh_pubkeys      = [
+    var.ssh_pubkey
+  ]
+
+  # ignore_x509_cn_check = true
+
+  worker_pool "suraj-aws-wp" {
+    count         = 1
+    # instance_type = var.workers_type
+    ssh_pubkeys   = [
+      var.ssh_pubkey
+    ]
+
+    # os_arch    = "arm64"
+    # os_channel = "alpha"
+  }
```

Comment:
We can work towards making this configuration as we want. Right now it is AWS AMD64 specific and has no worker pools or lokomotive components specified.

Reply:
Should they be configurable? At least for the benchmarked workers

Reply:
Added two worker pools as we discussed yesterday. We need to figure out how many nodes and how what type, etc. Added them as TODOs.